### PR TITLE
Telegram notify 1.0.4

### PR DESCRIPTION
**Pull request by mcaldas**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [X] __I will not move an already shared step version's tag to another commit__
- [X] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [X] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [X] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [X] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by mcaldas** [comment]
Hi, any update on this?

File: steps/telegram-notify/1.0.2/assets/icon.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
```
Comment (lpusok) [inline steps/telegram-notify/1.0.2/assets/icon.svg]: Icon should be on the path: `steps/telegram-notify/assets/icon.svg`; without version in the path.

Comment (mcaldas) [reply steps/telegram-notify/1.0.2/assets/icon.svg]: @lpusok im not sure where that came from, on my local repo i dont see any version path :/
The icon is on the root of the dir under assets :/
How can i change that?

Comment (lpusok) [reply steps/telegram-notify/1.0.2/assets/icon.svg]: It is in your steplib fork (created by bitrise :share command): https://github.com/mcaldas/bitrise-steplib/tree/telegram-notify-1.0.4/steps/telegram-notify

Comment (lpusok) [reply steps/telegram-notify/1.0.2/assets/icon.svg]: I recommend to try to start the share process again.

Comment (mcaldas) [reply steps/telegram-notify/1.0.2/assets/icon.svg]: @lpusok tried again and still got that icon stuck there :/
Any other suggestion?

Comment (lpusok) [reply steps/telegram-notify/1.0.2/assets/icon.svg]: Can you try to remove 1.0.2 directory from the forked steplib repo manually? Or remove the complete fork repo to be sure.

**Comment by lszucs** [comment]
Closing because https://github.com/bitrise-io/bitrise-steplib/pull/1893 superseeds this.

**Pull request closed without merging**